<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionMask_Directory"/>
    <addaction name="actionImport_Mask"/>
   </widget>
   <widget class="QMenu" name="menuAnnotation">
    <property name="title">
     <string>Annotation</string>
    </property>
    <addaction name="actionBrowse"/>
    <addaction name="actionPolygon"/>
    <addaction name="actionLivewire"/>
    <addaction name="actionBounding_Box"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionLine"/>
    <addaction name="actionDot"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuLable">
    <property name="title">
     <string>Lable</string>
    </property>
    <addaction name="actionImport_Label"/>
    <addaction name="actionExport_Lbale"/>
    <addaction name="actionLoad_Default"/>
    <addaction name="actionSet_as_Default"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionNext_Image"/>
    <addaction name="actionPrevious_Image"/>
    <addaction name="actionAuto_Contrast"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Annotations"/>
    <addaction name="actionClean_Noisy_Annotations"/>
    <addaction name="actionScreen_Shot"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionConfig"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnnotation"/>
   <addaction name="menuLable"/>
   <addaction name="menuTools"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionPrevious_Image"/>
   <addaction name="actionNext_Image"/>
   <addaction name="separator"/>
   <addaction name="actionBrowse"/>
   <addaction name="actionPolygon"/>
   <addaction name="actionLivewire"/>
   <addaction name="actionBounding_Box"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionLine"/>
   <addaction name="actionDot"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionSet_as_Default"/>
   <addaction name="actionLoad_Default"/>
   <addaction name="separator"/>
   <addaction name="actionClean_Noisy_Annotations"/>
   <addaction name="actionExport_Annotations"/>
   <addaction name="actionScreen_Shot"/>
   <addaction name="separator"/>
   <addaction name="actionAuto_Contrast"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../icons/open.png</normaloff>../icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>../icons/save.png</normaloff>../icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionBrowse">
   <property name="icon">
    <iconset>
     <normaloff>../icons/browse.png</normaloff>../icons/browse.png</iconset>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="icon">
    <iconset>
     <normaloff>../icons/polygon.png</normaloff>../icons/polygon.png</iconset>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionLivewire">
   <property name="icon">
    <iconset>
     <normaloff>../icons/livewire.png</normaloff>../icons/livewire.png</iconset>
   </property>
   <property name="text">
    <string>Livewire</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionBounding_Box">
   <property name="icon">
    <iconset>
     <normaloff>../icons/bbx.png</normaloff>../icons/bbx.png</iconset>
   </property>
   <property name="text">
    <string>Bounding Box</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="icon">
    <iconset>
     <normaloff>../icons/circle.png</normaloff>../icons/circle.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset>
     <normaloff>../icons/line.png</normaloff>../icons/line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionDot">
   <property name="icon">
    <iconset>
     <normaloff>../icons/dot.png</normaloff>../icons/dot.png</iconset>
   </property>
   <property name="text">
    <string>Dot</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionImport_Label">
   <property name="icon">
    <iconset>
     <normaloff>../icons/import.png</normaloff>../icons/import.png</iconset>
   </property>
   <property name="text">
    <string>Import Label</string>
   </property>
  </action>
  <action name="actionExport_Lbale">
   <property name="icon">
    <iconset>
     <normaloff>../icons/export.png</normaloff>../icons/export.png</iconset>
   </property>
   <property name="text">
    <string>Export Label</string>
   </property>
  </action>
  <action name="actionLoad_Default">
   <property name="icon">
    <iconset>
     <normaloff>../icons/load_label.png</normaloff>../icons/load_label.png</iconset>
   </property>
   <property name="text">
    <string>Load Default</string>
   </property>
  </action>
  <action name="actionSet_as_Default">
   <property name="icon">
    <iconset>
     <normaloff>../icons/pin_label.png</normaloff>../icons/pin_label.png</iconset>
   </property>
   <property name="text">
    <string>Set as Default</string>
   </property>
  </action>
  <action name="actionExport_Annotations">
   <property name="icon">
    <iconset>
     <normaloff>../icons/export_annotation.png</normaloff>../icons/export_annotation.png</iconset>
   </property>
   <property name="text">
    <string>Export Annotations</string>
   </property>
  </action>
  <action name="actionClean_Noisy_Annotations">
   <property name="icon">
    <iconset>
     <normaloff>../icons/clean_annotation.png</normaloff>../icons/clean_annotation.png</iconset>
   </property>
   <property name="text">
    <string>Clean Noisy Annotations</string>
   </property>
  </action>
  <action name="actionMask_Directory">
   <property name="icon">
    <iconset>
     <normaloff>../icons/mask.png</normaloff>../icons/mask.png</iconset>
   </property>
   <property name="text">
    <string>Mask Directory</string>
   </property>
  </action>
  <action name="actionImport_Mask">
   <property name="icon">
    <iconset>
     <normaloff>../icons/import.png</normaloff>../icons/import.png</iconset>
   </property>
   <property name="text">
    <string>Import Mask</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset>
     <normaloff>../icons/zoom_in.png</normaloff>../icons/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset>
     <normaloff>../icons/zoom_out.png</normaloff>../icons/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionNext_Image">
   <property name="icon">
    <iconset>
     <normaloff>../icons/next.png</normaloff>../icons/next.png</iconset>
   </property>
   <property name="text">
    <string>Next Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPrevious_Image">
   <property name="icon">
    <iconset>
     <normaloff>../icons/previous.png</normaloff>../icons/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>../icons/delete.png</normaloff>../icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionScreen_Shot">
   <property name="icon">
    <iconset>
     <normaloff>../icons/screenshot.png</normaloff>../icons/screenshot.png</iconset>
   </property>
   <property name="text">
    <string>Screen Shot</string>
   </property>
  </action>
  <action name="actionAuto_Contrast">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/contrast.png</normaloff>../icons/contrast.png</iconset>
   </property>
   <property name="text">
    <string>Auto Contrast</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="icon">
    <iconset>
     <normaloff>../icons/config.png</normaloff>../icons/config.png</iconset>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>../icons/about.png</normaloff>../icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
